fastexcel-core/pom.xml

```diff
@@ -6,16 +6,12 @@
     <parent>
         <groupId>cn.idev.excel</groupId>
         <artifactId>fastexcel-parent</artifactId>
-        <version>${revision}</version>
+        <version>1.1.0</version>
```

Comment:
Not quite sure how you guys currently release. But maven-release-plugin or version-maven-plugin can update the version at once.